### DISCUSSION TITLE
OpenHAB Intergation

### DISCUSSION THREAD

**Discussion by allanstevens** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#781] [Show and tell]
I’ve got mine working with OpenHAB now and thought my config might be helpful for others.

![image](https://github.com/user-attachments/assets/82df56cb-39ad-4fdd-a9a8-a66ac3ede2dd)

https://community.openhab.org/t/mqtt-whirlpool-control/124283/18?u=digitalgeek.me

**Comment by Snotmann** [comment]
@allanstevens what means heater red and heater green ?

**Reply by allanstevens** [reply]
Both switches will turn on the heater. Once it's on, Heater Green will indicate that it has reached the desired temperature, and Heater Red will indicate that it's still heating. I had hoped to have a single item with three states, but I can't figure out the transformation pattern for the thing. I'm hoping someone on the OpenHAB forums can offer some support.
